features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/apim_health_availability_alert_0.siddhi

```diff
@@ -47,6 +47,10 @@ define stream EmailAlertStream (type string, message string, alertTimestamp	stri
 
 define stream ApimAllAlertSummary (type string, tenantDomain string, message string, severity int, alertTimestamp long);
 
+Define stream ApimApiHealthMonitorAlertStream (apiName string,apiVersion string,apiCreator string, apiCreatorTenantDomain string, message string, severity int, alertTimestamp long);
```

Comment:
@ruwiniwj , The defined stream is different from the one used in join query.

Reply:
Fixed with https://github.com/wso2/analytics-solutions/pull/253